interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+			<input id="NEURO11" name="NEURO11" type="checkbox" value="on" >
+			<label for="NEURO11"> <?php echo xlt('Tingling');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO12" name="NEURO12" type="checkbox" value="on" >
+			<label for="NEURO12"> <?php echo xlt('Numbness');?></label>
+		</span>
+		</TD>
+	
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="NEURO_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+		
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('HEME:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="HEME1" name="HEME1" type="checkbox" value="on" >
+			<label for="HEME1"> <?php echo xlt('Bleeding');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="HEME2" name="HEME2" type="checkbox" value="on" >
+			<label for="HEME2"> <?php echo xlt('Bruising');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="HEME3" name="HEME3" type="checkbox" value="on" >
+			<label for="HEME3"> <?php echo xlt('Anemia');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="HEME4" name="HEME4" type="checkbox" value="on" >
+			<label for="HEME4"> <?php echo xlt('Sickle Cell Disease');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="HEME5" name="HEME5" type="checkbox" value="on" >
+			<label for="HEME5"> <?php echo xlt('Hx Blood Transfusion');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="HEME6" name="HEME6" type="checkbox" value="on" >
+			<label for="HEME6"> <?php echo xlt('HIV+');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="HEME_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+	
+	<TR class="alt">
+	<TD>
+			<B><?php echo xlt('ENDO:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="ENDO1" name="ENDO1" type="checkbox" value="on" >
+			<label for="ENDO1"> <?php echo xlt('Polydipsia');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENDO2" name="ENDO2" type="checkbox" value="on" >
+			<label for="ENDO2"> <?php echo xlt('Heat Intolerance');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENDO3" name="ENDO3" type="checkbox" value="on" >
+			<label for="ENDO3"> <?php echo xlt('Cold Intolerance');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENDO4" name="ENDO4" type="checkbox" value="on" >
+			<label for="ENDO4"> <?php echo xlt('Excessive Sweating');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENDO5" name="ENDO5" type="checkbox" value="on" >
+			<label for="ENDO5"> <?php echo xlt('Feelings of Weakness');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
```

Comment:
I always got shot down for punctuation in the information to be translated it breaks the translation